ospfd/ospf_packet.h

```diff
@@ -34,10 +34,10 @@
 #define OSPF_LS_ACK_MIN_SIZE      0U
 
 #define OSPF_MSG_HELLO         1  /* OSPF Hello Message. */
-#define OSPF_MSG_DB_DESC       2  /* OSPF Database Descriptoin Message. */
+#define OSPF_MSG_DB_DESC 2	/* OSPF Database Description Message. */
```

Comment:
Please keep formatting consistent with others or change others as well.

Reply:
@ton31337 Sorry for that...  `git-clang-format` only formatted those two lines... Just force-pushed. Thanks